source_code/src/FLASH/flash_mem.c

```diff
@@ -42,7 +42,9 @@
 void memoryBoundaryErrorCallback(void)
 {
     // We'll add more debug later if needed
+#ifndef MINI_BOOTLOADER
     usbPutstr("#MBE");
+#endif
```

Comment:
but how come on atmel studio, which doesn't compile the usb core, we don't get any error?